_episodes/17-live.md

```diff
@@ -87,6 +87,22 @@ important to keep in mind, however, that feedback is not helpful if you cannot u
 Many error messages are obscure and not written with novices in mind. Continue to use the strategies for error framing that
 [we learned earlier]({{ page.root }}{% link _episodes/08-motivation.md %}) to make sure this feedback is useful to learners.
 
++ **Handling errors in coding as a basic programming skill** – one way of encouraging learners to deal with coding errors that
+may occur in their own work is to introduce them to the idea of a “bug”. Properly identifying possible “bugs” (errors) in a code
+and “debugging” them (finding a solution to remove/resolve them) is a unique skill on its own. It is not too distant to what could 
+be considered as “proofreading” of a written text, from finding and hopefully fixing issues as simple as spelling mistakes or character 
+omissions (syntax errors) up to as complex as wrong logical sequences (logical errors) or incorrect calculations (arithmetic errors). 
+However, “debugging” goes a little bit beyond that, as not only a good grasp of specific coding language principles is needed, but also
+experience in reading the feedback software that executes the code can provide about encountered problems (error messages).
+
++ **Dealing with errors, when to (actually) start?** - A few minutes of a lesson can be spent on intentionally making a few basic errors 
+to explain the idea and to familiarise the learners with error messages and how to read them. The simplest and reproducible errors are 
+definitely syntax errors; they are also ones that learners will most likely encounter first and may start to spot on their own. Syntax 
+errors also have relatively well-defined error messages in the majority of interpreting software. However, explaining debugging beyond
+this point should be done only when opportunities arise. Own coding mistakes and errors made by learners when either copying the instructor
+or while working on their own should be considered a teaching/learning resource, giving a chance to digress about more specific issues and
+how to deal with them when writing or reviewing the code.
```

Comment:
```suggestion
+ **Intentional vs accidental errors** - It can be useful to intentionally demonstrate common mistakes and error messages. Alternatively, mistakes made by the Instructor or learners can offer useful opportunities to learn about and positively frame errors.
```